_draft-tests/SC3-1-1-html-lang-valid.md

```diff
@@ -34,7 +34,7 @@ success_criterion:
 ```
 
 ```html
-<html xml:lang="xyz" lang="fr">
+<html xml:lang="xyz" lang="xyz">
```

Comment:
There should be a test case that checks that xml:lang is ignored when lang is set. This doesn't do that.

Reply:
That is the last case in the passed scenario. (The xml:lang is ignored with lang is set).

Although you are right. I have amended this fail scenario like so:
```html
<html xml:lang="en" lang="xyz">
```